### PR TITLE
Add schedule dialog

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1078
[Problem] - Schedule dialog doesn't exists
[Solution]
 - Add dialog element to Sample App
 - Connect Pages using it

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/mobile/SchedulingApp/js/app.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+			page = document.getElementsByClassName("ui-page-active")[0];
+			pageId = page ? page.id : "";
+
+			if (pageId === "main" && !activePopup) {
+				try {
+					tizen.application.getCurrentApplication().exit();
+				} catch (ignore) {
+				}
+			} else {
+				window.history.back();
```
Comment (DohyungLim) [inline examples/mobile/SchedulingApp/js/app.js]: Use tau.history.back rather than window.history.back.

File: examples/mobile/SchedulingApp/js/app.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+			}
+		}
+	});
+}());
```
Comment (DohyungLim) [inline examples/mobile/SchedulingApp/js/app.js]: add end-line.

File: examples/mobile/SchedulingApp/index.html [hunk 3]
```diff
@@ -20,12 +20,35 @@
...
-
+			<div class="ui-popup" id="turn-on-dialog">
+				<div class="ui-popup-header">
+					Add Schedule
+				</div>
+				<div class="ui-popup-content">
+					<ul class="ui-listview ui-content-area">
+						<li>
+							<div class="ui-li-text">
+								<a href="./create-event.html" class="ui-li-text-title">Turn device on</a>
```
Comment (DohyungLim) [inline examples/mobile/SchedulingApp/index.html]: Is it intended to go all to create-event.html?

**Review by DohyungLim — COMMENTED** [review]
3 comment.

File: examples/mobile/ScheduleApp/.tproject [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (singa2000) [inline examples/mobile/ScheduleApp/.tproject:1]: Is this file needed?
I think it's no needed for examples.

File: examples/mobile/ScheduleApp/css/style.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0 auto;
+    padding: 0;
+    background-color: #222222;
+    color: #ffffff;
+}
+.page {
```
Comment (TomaszLukawskiSam) [inline examples/mobile/ScheduleApp/css/style.css]: These styles seems not related to the app. I don't see `page` class, `content-text` id, `contents` class in index.hrml

File: examples/mobile/ScheduleApp/css/style.css [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    height: 100%;
+    display: table;
+}
+.contents {
+    display: table-cell;
+    vertical-align: middle;
+    text-align: center;
+    -webkit-tap-highlight-color: transparent;
+}
+#content-text {
```
Comment (singa2000) [inline examples/mobile/ScheduleApp/css/style.css]: Is it right "#"? Is it your intended?

File: examples/mobile/ScheduleApp/.tproject (hunk 4 above)
Comment (lslachciak) [reply examples/mobile/ScheduleApp/.tproject:1]: @singa2000 I requested this from Kornelia in previous PR. All examples have sample .tproject file so we can test them on emulator.

File: examples/mobile/ScheduleApp/css/style.css (hunk 5 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: Ok, this style file was generated by Tizen Studio, I will remove it.

File: examples/mobile/ScheduleApp/css/style.css (hunk 6 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: This CSS file was generated by Tizen Studio, I will remove it

File: examples/mobile/ScheduleApp/css/style.css (hunk 5 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: I removed it

File: examples/mobile/ScheduleApp/css/style.css (hunk 6 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: I removed entire file.

**Comment by lslachciak** [comment]
Some comments regarding add schedule dialog

image

Can we eliminate easily following items? 
- horizontal lines on dialog
- dialog items underline

Reference guide don't have them

File: examples/mobile/ScheduleApp/index.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+				<button class="ui-btn ui-btn-icon ui-btn-icon-more" data-style="flat"></button>
+			</div>
+		</header>
+		<div class="ui-content">
+			<div class="ui-popup" id="turn-on-dialog">
+				<div class="ui-popup-header">
+					Add Schedule
+				</div>
+				<div class="ui-popup-content">
+					<ul class="ui-listview ui-content-area">
```
Comment (lslachciak) [inline examples/mobile/ScheduleApp/index.html:33]: resigning fromui-content-area should help with horizontal lines visible

**Review by lslachciak — APPROVED** [review]
works well
